### ISSUE TITLE
HTML editing features don't work well in PHP files

### ISSUE THREAD

**Issue by OhligerJ** [microsoft/vscode#62077]
- VSCode Version: Version: 1.28.2 (user setup)
Commit: 7f3ce96ff4729c91352ae6def877e59c561f4850
Date: 2018-10-17T00:23:51.859Z
Electron: 2.0.9
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
- OS Version: Windows 7

Steps to Reproduce:

1. Create new PHP file
2. Type in div.class, hit tab, hit enter

Current Solution:

Change file type to HTML, and then back to PHP

Proposed solution: 

In a file type where HTML tab-completion works, HTML auto-indenting should also work

Does this issue occur when all extensions are disabled?: Quick Google search did not provide clear instructions on how to perform this in Windows.

**Comment by AeonFr** [comment]
I was just about to report the same issue.

The tab autocompletion on PHP is so buggy it's practically useless to have it on.

Here are two videos that show my struggles trying to do basic things. VSCode is not only _not doing what it's supposed to do_, it's also __completely broken everything else in the process__.

![image](https://user-images.githubusercontent.com/11742754/57971366-51b9ee00-798d-11e9-929a-c70aa550b4e4.png)

![image](https://user-images.githubusercontent.com/11742754/57971370-5b435600-798d-11e9-94ac-e2406601e140.png)

**Comment by TomS-** [comment]
This is definitely a bug, the fact that it's been around for so long is a little concerning. I've been quite about this for a while. But it's really starting to annoy me. It's a shame because VSCode is amazing code editor otherwise.

**Comment by baileyherbert** [comment]
Yep. I don't recall have many issues with PHP+HTML up until now, but on the latest builds it is impossible to use VSCode when working with HTML inside PHP files. The auto-indent is completely broken.

**Comment by mbekir705-pnc** [comment]
in september 2020 still same issue is exist. Isn't there any solution, really?

**Comment by asccc** [comment]
Using `"editor.autoIndent": "advanced",` instead of `"editor.autoIndent": "full",` helped as a workaround. 
Its not perfect but at least you'll be able to close HTML tags without re-indenting them every time.

**Comment by saminton** [comment]
#103147 reported this a while ago and got a rather unsatisfying response... I have yet to find a solution.

**Comment by roblourens** [comment]
Merging into https://github.com/microsoft/vscode/issues/62077
